mmv1/third_party/terraform/services/identityplatform/resource_identity_platform_config_test.go

```diff
@@ -0,0 +1,126 @@
+package identityplatform_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+)
+
+func TestAccIdentityPlatformConfig_update(t *testing.T) {
+	acctest.SkipIfVcr(t)
```

Comment:
```suggestion
	# acctest.SkipIfVcr(t)
```
We can temporally comment out this line so the update test will be run in VCR and you'll be able to see test result as well, and add the skip back before merging the PR.